meteor/server/api/blueprints/context/adlibActions.ts

```diff
@@ -99,6 +100,9 @@ export class ActionExecutionContext extends ShowStyleContext implements IActionE
 	private readonly rundownPlaylist: RundownPlaylist
 	private readonly rundown: Rundown
 
+	public readonly notes: RawNote[] = []
+	private readonly blackHoleNotes: boolean
```

Comment:
minor note: this is temporary, right? perhaps rename it to `tempSendNotesIntoBlackHole`?